### ISSUE TITLE
Missing possibility to edit a town

### ISSUE THREAD

**Issue by nabilys** [organicmaps/organicmaps#7802]
⚠ Have you searched for similar, already existing issues?

**Describe the issue**
When I tap on a town in the map, I don’t have the possibility to edit it. In my example, the Wikipedia link is wrong and I can’t change it

**Steps to reproduce**
1. Tap in a town on the map

**Expected behaviour**
Show the possibility to edit it

**Screenshots**
![IMG_6796](https://github.com/organicmaps/organicmaps/assets/59312735/7b951584-bbe7-4d61-95f8-ab710d0cfac7)

**System information:**
 - Operating system and its version: ios 17.4
 - Organic Maps version: 2024.03.31
 - Device Model: iPhone 14 Pro

**Comment by map-per** [comment]
Some features, like towns, can not be edited in Organic Maps (OM). 
This is, because OM targets at unexperienced users. So editing towns is just considered to be too dangerous. (And to some extend this is also to prevent vandalism.)

But apart from that you can not edit wikipedia entries in OM anyway. But as all of OMs map data comes from OpenStreetMap you can fix the data there (see: https://wiki.openstreetmap.org/wiki/Beginners%27_guide)

**Comment by biodranik** [comment]
Editing wiki articles anywhere should be feasible. Is there an issue for that?

**Comment by map-per** [comment]
No, Wikipedia is commented out in [editor.config](https://github.com/organicmaps/organicmaps/blob/fff96447ff937eca61b33705304b92bb8b0f3833/data/editor.config#L110C1-L110C79), so it's not editable.

**Comment by biodranik** [comment]
It is commented because the edit cell in Android and iOS is not ready yet.
